app/controllers/slack_authorizations_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class SlackAuthorizationsController < ApplicationController
+  before_action :authorize_project_admin
+
+  def create
+    Slack::EntityCreator.new(request.env["omniauth.auth"], request.env["omniauth.strategy"].access_token, request.env["omniauth.params"]).call
```

Comment:
You search for project in this controller and inside creator. Probably a good idea to pass project from controller to service

Reply:
@evheny0  It's probably good for now. I wanted to pass the whole params hash